lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImpl.java

```diff
@@ -123,6 +123,14 @@ public void shutdown() {
         shutdownAllEngines();
     }
 
+    public List<LocationListener> getListeners() {
```

Comment:
Ok now I understand the confusion. These tests were meant to test listeners registered by LOST on the framework `LocationManager` (not by client apps on LOST). See comments below.